cmd/talosctl/cmd/mgmt/cluster/create.go

```diff
@@ -933,6 +954,13 @@ func init() {
 	createCmd.Flags().IntVar(&controlPlanePort, controlPlanePortFlag, constants.DefaultControlPlanePort, "control plane port (load balancer and local API port)")
 	createCmd.Flags().BoolVar(&dhcpSkipHostname, "disable-dhcp-hostname", false, "skip announcing hostname via DHCP (QEMU only)")
 	createCmd.Flags().BoolVar(&skipBootPhaseFinishedCheck, "skip-boot-phase-finished-check", false, "skip waiting for node to finish boot phase")
+	createCmd.Flags().BoolVar(&networkChaos, networkChaosFlag, false, "enable to use network chaos parameters when creating a qemu cluster")
+	createCmd.Flags().DurationVar(&jitter, jitterFlag, 5*time.Millisecond, "specify jitter on the bridge interface when creating a qemu cluster (default: 5ms)")
+	createCmd.Flags().DurationVar(&latency, latencyFlag, 35*time.Millisecond, "specify latency on the bridge interface when creating a qemu cluster (default: 35ms)")
+	createCmd.Flags().Float64Var(&packetLoss, packetLossFlag, 0.0, "specify percent of packet loss on the bridge interface when creating a qemu cluster. e.g. 50% = 0.50 (default: 0.0)")
```

Comment:
Changed to float so user can set the value as a fraction. e.g. 50% would be input as .50